modules/support-on-various-platforms.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support-stackrox.adoc
+:_module-type: CONCEPT
+[id="support-on-various-platforms_{context}"]
+= Support on various platforms
+
+[role="_abstract"]
+StackRox support team supports a platform version based on the upstream (or vendor) product’s support lifecycle for that version. When a platform version support reaches its end-of-life (EOL) or is not actively maintained, StackRox no longer support it.
```

Comment:
"The StackRox support team supports"

Reply:
"When a platform version support" doesn't make sense to me. Maybe, "When support of a platform reaches..."?

s/end-of-life/end of life (The hyphens are only needed if adjectivally describing a noun, which in this case it's not 😁 )
s/no longer support it/no longer supports it